xdsl/irdl.py

```diff
@@ -353,23 +362,23 @@ def __init__(self, typ: Attribute | type[Attribute] | AttrConstraint):
         self.constr = attr_constr_coercion(typ)
 
 
-Operand = Annotated[SSAValue, OperandDef]
+Operand: TypeAlias = SSAValue
 
 
 @dataclass(init=False)
 class VarOperandDef(OperandDef, VariadicDef):
     """An IRDL variadic operand definition."""
 
 
-VarOperand = Annotated[list[SSAValue], VarOperandDef]
+VarOperand: TypeAlias = list[SSAValue]
 
 
 @dataclass(init=False)
 class OptOperandDef(VarOperandDef, OptionalDef):
     """An IRDL optional operand definition."""
 
 
-OptOperand = Annotated[SSAValue | None, OptOperandDef]
+OptOperand: TypeAlias = SSAValue | None
```

Comment:
what does this do? Why does `VarOpResult` not need it?

Reply:
I'll add it to the others as well, it's just an explicit type annotation otherwise!